recipes/FigureGenerator/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "FigureGenerator" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FigureGenerator-{{ version }}.tar.gz
+  sha256: 311cfed9f8e09037e9fa8d5bc0127c33264c4fa375595d526f6119e6db582fb0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - black
+    - coverage
+    - ipywidgets
+    - numpy ==1.19.2
```

Comment:
is that really necessary? it will be incompatible with lots of packages

Reply:
This is fixed in the latest version [[ref](https://github.com/CBICA/FigureGenerator/blob/main/setup.py#L23)].